solidity/script/DeployRouter7683.s.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.25 <0.9.0;
+
+import { Script } from "forge-std/Script.sol";
+import { console2 } from "forge-std/console2.sol";
+
+import { TypeCasts } from "@hyperlane-xyz/libs/TypeCasts.sol";
+import { GasRouter } from "@hyperlane-xyz/client/GasRouter.sol";
+import { TransparentUpgradeableProxy } from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import { ProxyAdmin } from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+import { Router7683 } from "../src/Router7683.sol";
+
+/// @dev See the Solidity Scripting tutorial: https://book.getfoundry.sh/tutorials/solidity-scripting
+contract DeployRouter7683 is Script {
+    function run() public {
+        uint256 deployerPrivateKey = vm.envUint("DEPLOYER_PK");
+
+        string memory ROUTER_SALT = vm.envString("ROUTER7683_SALT");
+        address mailbox = vm.envAddress("MAILBOX");
+        address permit2 = vm.envAddress("PERMIT2");
+        address proxyAdminOwner = vm.envOr("PROXY_ADMIN_OWNER", address(0));
+        address owner = vm.envAddress("ROUTER_OWNER");
+        uint256[] memory domains = vm.envUint("DOMAINS", ",");
+        uint32[] memory _domains = new uint32[](domains.length);
+        bytes32[] memory routers = new bytes32[](domains.length);
+        GasRouter.GasRouterConfig[] memory gasConfigs = new GasRouter.GasRouterConfig[](domains.length);
+
+        vm.startBroadcast(deployerPrivateKey);
+
+        ProxyAdmin proxyAdmin = new ProxyAdmin();
```

Comment:
I know i was a stickler for proxied contracts for the other bounties, but in the context of 7683, proxies seem unnecessary, so maybe simpler to not proxy?

Reply:
If we don't use a proxy then it would be a good idea to remove the `initialize` function and move the the call to `_MailboxClient_initialize` to the constructor. Do you agree?